gui/gamesetup/gamesetup~!extra_commands.js

```diff
@@ -1,4 +1,4 @@
-
+var linkidShort = Date.now().toString().substring(10);
```

Comment:
Move inside the `g_NetworkCommands["/jitsi"]`

Reply:
its global. because the link should stay the same during the config and the game. its generated from milisecond from now.